src/voice_missed_call_service.js

```diff
@@ -38,7 +39,8 @@ go.app = function() {
 
         // EndState
         self.add('state_end', function(name) {
-            return new EndState(name, {text: ''});
+            // freeswitch crashes if the text is empty
+            return new EndState(name, {text: 'Thank you.'});
```

Comment:
I think for freeswitch, we can have the text as `.`, which translates to a pause in the text to speech

Reply:
Thanks, let me test that.

Reply:
@rudigiesler That also made freeswitch crash, maybe it only works if it is part of more text. The `Thank you` should be fine, the whole call just lasts 2 seconds.